web/templates/settings/bot_avatar_row.hbs

```diff
@@ -8,8 +8,10 @@
                 <button type="submit" class="btn open_edit_bot_form tippy-zulip-delayed-tooltip" data-sidebar-form="edit-bot" data-tippy-content="{{t 'Edit bot' }}" data-email="{{email}}">
                     <i class="fa fa-pencil blue" aria-hidden="true"></i>
                 </button>
-                <a type="submit" download="{{zuliprc}}" class="btn download_bot_zuliprc tippy-zulip-delayed-tooltip" data-tippy-content="{{t 'Download zuliprc' }}" data-email="{{email}}">
-                    <i class="fa fa-download sea-green" aria-hidden="true"></i>
+                <a tabindex="-1" download="{{zuliprc}}" class="download_bot_zuliprc tippy-zulip-delayed-tooltip no-underline" data-tippy-content="{{t 'Download zuliprc' }}" data-email="{{email}}">
```

Comment:
You can remove the `tabindex` attribute from here. It's not needed anymore as the button tag is there.

Reply:
@sbansal1999 I added that tabindex attribute, because I realized there are moments that the anchor tag is focused in the koeyboard navigation (not always) and it breaks the styles of the button. So, adding the negative tabindex the anchor tab is disabled in the keyboard navigation.

Should I delete that tabindex anyways?

Thanks for the info! I'll squash the commits into a single one.

Reply:
> I realized there are moments that the anchor tag is focused in the koeyboard navigation (not always) and it breaks the styles of the button

I tried removing the attribute and it worked for me. Can you describe the exact steps to reproduce this?

Reply:
I tried to reproduce it, but It does not focus the anchor tag anymore. Maybe It happened with some changes I did before adding the button tag. Anyways, this last commit Fixes #26198.  

I also squashed the 4 commits into 1. The Git guide was useful and I learnt something new. Thanks!

Have a nice day! :D